Examples/Python/tests/test_examples.py

```diff
@@ -1223,7 +1223,7 @@ def test_ML_Ambiguity_Solver(tmp_path, assert_root_hash):
     env["ACTS_LOG_FAILURE_THRESHOLD"] = "ERROR"
     try:
         subprocess.check_call(
-            [sys.executable, str(script), "-n1", "--MLSolver"],
+            [sys.executable, str(script), "-n1", "--SolverType ML"],
```

Comment:
this needs to be updated after the flag changed

Reply:
I have updated the flag and error has been fixed. The test is not failing anymore.